### ISSUE TITLE
Raiden on Arbitrum: fix timeouts naming convention

### ISSUE THREAD

**Issue by netcriptus** [raiden-network/raiden-contracts#1580]
As pointed in [this](https://github.com/raiden-network/raiden-contracts/pull/1573#discussion_r769433236) and [this](https://github.com/raiden-network/raiden-contracts/pull/1573#discussion_r769438688) comments, there's some confusion regarding the naming of timeouts in the Smart Contracts.

There are 2 concepts of time going around, one for absolute timestamp, and another one for number of seconds an event, and the names do not reflect it at all.

Refactor the contracts with better naming, and unifying names that should reflect the same thing.

**Comment by andrevmatos** [comment]
My suggestion would be be `_timestamp` for absolute points in time, and `_timeout` for relative time deltas, but I don't have strong feelings for any particular naming schema, as long as it's consistent.

I think we should take the time to do this renaming in #1573 , since that's handling exactly this change, so we avoid even polluting commit history with names that aren't descriptive.

**Comment by weilbith** [comment]
@andrevmatos and I have discussed this topic now and settled on a scheme.
There are two different concepts of the timing functionality here for 3 cases. All of them specify a relative time value that gets added on top of the current time when an event happens. So for example a time value for settling a channel, that gets added to the timestamp when a channel gets closed.
The individual future timestamp that gets calculated has then two different meanings, depending on the context. In the case of the channel settle part, it means the channel becomes settleable after this timestamp has passed. In another case like for withdrawing funds from a channel it means it can be withdrawn until this timestamp has passed. It is our desire to express this significant difference in the code and make it very easy to read and understand without much guessing.

Thereby we ended up with calling these variables by the following concept:
- `*_timeout` (e.g. `settle_timeout`) - relative time value that gets added to event timestamp to get any of the following values
- `*able_until` (e.g. `withdrawable_until`) - absolute timestamp until which it is possible to execute a specific action
- `*able_after` (e.g. `settleable_after`) - absolute timestamp only after which it is possible to execute a specific action
